### DISCUSSION TITLE
User research: discuss and agree on general approach and aims

### DISCUSSION THREAD

**Discussion by shonfeder** [informalsystems/quint#200] [General]
We know we'll want to do more user research -- beyond the initial user interviews we've done and the copious info we've gathered from our first- and second-hand experience with formalization.

afaiu, there are many different approaches to user research and communication. I think it  be helpful for us to ensure we have settled on a coherent and mutually understood approach before investing time in executing the research.

As an example, many UX people seem to warn against "asking users what they want". E.g.,

- https://uxplanet.org/stop-asking-users-what-they-want-21e9ba646bce
- https://www.nngroup.com/articles/first-rule-of-usability-dont-listen-to-users/

Do we want to head that warning, or do we have good reasons to ignore it?
 
In general, perhaps the discussion should begin by articulating and considering some questions like:

- what is our overall aim in soliciting user input ?
- what approaches to gathering user input will serve our aims?

**Comment by shonfeder** [comment]
Re: approaches, I am particularly interested in https://www.sciencedirect.com/topics/computer-science/participatory-design

> Participatory design entails user participation in design for work practice. Participatory design is a [democratic process](https://www.sciencedirect.com/topics/computer-science/democratic-process) for design (social and technological) of systems involving human work, based on the argument that users should be involved in designs they will be using, and that all stakeholders, including and especially users, have equal input into interaction design

**Comment by konnov** [comment]
I think our perspective on that has changed a lot. I don't think there is a need for this broad exploration anymore.
